.travis.yml

```diff
@@ -1,12 +1,8 @@
 language: php
 
 php:
-  - 5.3
```

Comment:
Yeah, but this is erroneous.

You did the same on `admin-bundle` and I had to add a missing test case: https://github.com/sonata-project/SonataAdminBundle/commit/03f2dc3445c11fbc60969aada7ee9fcb2bd17304

And I think another one is missing.

Please leave this part as is.